usaspending_api/awards/management/commands/load_subawards_sql/150_reset_unlinked_award_subaward_fields.sql

```diff
@@ -12,8 +12,13 @@ set
     total_subaward_amount = null,
     subaward_count = 0
 
+from
+    awards a
+    left outer join subaward s on s.award_id = a.id
```

Comment:
Is the fundamental difference that this filters on the join instead of the where?

Reply:
Actually, won't this just select every award, since there will always be a sub-award which isn't one of that award's subs?